content/blog/esptool-v5/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "esptool: Updates about the upcoming v5 major release"
+date: 2025-04-01T12:05:21+01:00
```

Comment:
This is an older version of the article, please wait with the merge until we update this.